src/LinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -114,7 +120,7 @@ LibunwindstackResult LibunwindstackUnwinderImpl::Unwind(
 
   unwindstack::Unwinder unwinder{max_frames, maps, &regs, memory};
   // Careful: regs are modified. Use regs.Clone() if you need to reuse regs later.
-  unwinder.Unwind();
+  unwinder.Unwind(nullptr, nullptr, absolute_address_to_size_of_functions_to_stop_at_);
```

Comment:
Parameter names in the form `/*parameter_name=*/` for the two `nullptr` might be useful here.

Reply:
Done.